### PR TITLE
[Translate] add the indonesian translation

### PR DESCRIPTION
**Pull request by dindinG41TR3**
I have translated Gcolor3 into Indonesian. hope can help warm regards from me indonesia enthusiast GNU/Linux :).

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # French translation for Gcolor3.
 # Copyright (C) 2013-2015 Nicolas Gaulard-Querol <nicolas.gquerol@gmail.com>
-# Copyright (C) 2015-2016 Timothée Floure <timothee.floure@fnux.ch>
```
Comment (Fnux) [inline po/fr.po]: Looks like you don't like me ;(

It doesn't make any sense, can you fix it ?

File: po/ChangeLog [hunk 2]
```diff
@@ -1,3 +1,6 @@
+2013-06-08: v0.1: Added Dutch translation
```
Comment (Fnux) [inline po/ChangeLog]: It doesn't make any sense either.

Be careful when you commit! You can use `git add -p` and `git commit -v` to review your changes before committing.

File: po/fr.po (hunk 1 above)
Comment (dindinG41TR3) [reply po/fr.po]: forgive me i tk accidentally deleted it. well i will correction

File: po/ChangeLog (hunk 2 above)
Comment (dindinG41TR3) [reply po/ChangeLog]: forgive me I accidentally deleted it. well i will correction

Comment (dindinG41TR3) [reply po/ChangeLog]: i was fix it

**Comment by dindinG41TR3** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  dindin_hernawan@yahoo.com
    Domain dev.xentaos.org has exceeded the max emails per hour (5/5 (100%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o9.sgmail.github.com ([167.89.101.2]:38176)
	by mejing.idwebhost.com with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <bounces+848413-bedc-root=dev.xentaos.org@sgmail.github.com>)
	id 1eNzxJ-001W6u-Ih
	for root@dev.xentaos.org; Sun, 10 Dec 2017 18:39:37 +0700
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=05Khnnwk4rr7JUpDEPsdZ0wh6aQ=; b=OehSkeAYucO06A5b
	fA/NolD9bQMYGbcI1Dxu/hNdq7rFzyYYGKM3arIK+OkM6gltnqFVgljLOLLemWwT
	lQcHW2xmFPvRPX59m3QFi+puXA/BwzMpevDMh4iCbNFACxqJnle1WidqSwuYUzYG
	hKVEBFeAunRqS8ya6sejk054Wq0=
Received: by filter0836p1mdw1.sendgrid.net with SMTP id filter0836p1mdw1-10857-5A2D1CCD-12
        2017-12-10 11:38:53.60235273 +0000 UTC
Received: from github-smtp2b-ext-cp1-prd.iad.github.net (github-smtp2b-ext-cp1-prd.iad.github.net [192.30.253.17])
	by ismtpd0004p1iad2.sendgrid.net (SG) with ESMTP id rA5V4F6hTuKlusc8-rd_Ug
	for <root@dev.xentaos.org>; Sun, 10 Dec 2017 11:38:53.479 +0000 (UTC)
Date: Sun, 10 Dec 2017 11:38:53 +0000 (UTC)

File: po/ChangeLog (hunk 2 above)
Comment (dindinG41TR3) [reply po/ChangeLog]: forgive by accidentally deleting it

Comment (dindinG41TR3) [reply po/ChangeLog]: maafkan saya

Comment (dindinG41TR3) [reply po/ChangeLog]: I'm sorry, I'm careless

**Comment by dindinG41TR3** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  dindin_hernawan@yahoo.com
    Domain dev.xentaos.org has exceeded the max emails per hour (6/5 (120%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
------ The body of the message is 21416 characters long; only the first
------ 5000 or so are included here.
Received: from o1.sgmail.github.com ([192.254.114.176]:41927)
	by mejing.idwebhost.com with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <bounces+848413-bedc-root=dev.xentaos.org@sgmail.github.com>)
	id 1eNzyA-001WC9-O6
	for root@dev.xentaos.org; Sun, 10 Dec 2017 18:40:31 +0700
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=7XMzb/pRSqmfod9ytmw5rACu/h0=; b=IeGw14Htg1nOVfwE
	10vLcauwl6DRfgKg9Giy+7YxbHcUH49PxN2OnxdAnhuxX7YBZkviKxhGrp5k1ROs
	y+gota0MrHDhOPGXU1LGkaUokYi2rHqkjEgMI9iloVj80nSE0US15taOh4n8Ghh9
	aXFW416kBxNCFEzAP0OyzrAZHhs=
Received: by filter0425p1mdw1.sendgrid.net with SMTP id filter0425p1mdw1-13819-5A2D1D03-2
        2017-12-10 11:39:47.069710113 +0000 UTC
Received: from github-smtp2b-ext-cp1-prd.iad.github.net (github-smtp2b-ext-cp1-prd.iad.github.net [192.30.253.17])
	by ismtpd0018p1iad2.sendgrid.net (SG) with ESMTP id 3PVtNxkIQDGcUtXY9u1pbw
	for <root@dev.xentaos.org>; Sun, 10 Dec 2017 11:39:46.991 +0000 (UTC)
Date: Sun, 10 Dec 2017 11:39:47 +0000 (UTC)

**Pull request closed without merging**